Tasks/SqlAzureDacpacDeploymentV1/README.md

```diff
@@ -71,9 +74,11 @@ The parameters of the task are described in details, including examples, to show
 
 - **Service Principal** : Uses the Authentication data from **Azure Subscription**. 
 
+- **Windows Authentication** : Uses the credentials of the self-hosted pipeline agent.
+
 **SQL DB Details**
 
-- **Azure SQL Server Name\*:** The connection string for the Azure SQL Server and the format is same as that is followed in SQL Server Management Studio. For example, FabrikamSQL.database.windows.net, 1433 or FabrikamSQL.database.windows.net are both valid Azure SQL Server names.
+- **SQL Server Name\*:** The connection string for the Azure SQL Server and the format is same as that is followed in SQL Server Management Studio. For example, FabrikamSQL.database.windows.net, 1433 or FabrikamSQL.database.windows.net are both valid Azure SQL Server names.
 
 - **Database Name\*:** The name of the Azure SQL Database like FabrikanDB. The Database will be created new if it does not exist, else it will be updated if it already exists.
```

Comment:
same "or" comment here - should probably clarify to "The name of the Azure SQL Database or SQL Server database like FabrikanDB. The database will be created new..."